hassio/addons/utils.py

```diff
@@ -40,6 +40,10 @@ def rating_security(addon):
     if addon.host_network:
         rating += -1
 
+    # Insecure PID namespace
+    if addon.host_pid:
+        rating += -1
```

Comment:
`-2` if the add-on run in same namespace, a attacker can access to host process.

Reply:
How about just setting the `rating = 1` just like with the docker API?